src/signals/IncidentMap/components/IncidentLayer.tsx

```diff
@@ -0,0 +1,91 @@
+/* SPDX-License-Identifier: MPL-2.0 */
+/* Copyright (C) 2022 Gemeente Amsterdam */
+
+import type { FeatureCollection, Feature as GeoJSONFeature } from 'geojson'
+import type { LatLngTuple } from 'leaflet'
+import type { Geometrie } from 'types/incident'
+import type { ReactElement } from 'react'
+
+import { useFetch } from 'hooks'
+import { useEffect, useState } from 'react'
+import useBoundingBox from 'signals/incident/components/form/MapSelectors/hooks/useBoundingBox'
+import configuration from 'shared/services/configuration/configuration'
+import L from 'leaflet'
```

Comment:
vind het iets omschrijvender om Leaflet te gebruiken ipv L

Reply:
zal ik in de volgende PR wel even aanpassen.